ansible/roles/operator-pipeline/templates/openshift/pipelines/community-hosted-pipeline.yml

```diff
@@ -305,6 +305,60 @@ spec:
         - name: remove-matching-namespace-labels
           value: "true"
 
+    # set the GitHub status to success if the pipeline passes
+    - name: set-github-status-success
+      when:
+        - input: "$(tasks.status)"
+          operator: in
+          values: ["Succeeded", "Completed"]
+      taskRef:
+        name: set-github-status
+      params:
+        - name: pipeline_image
+          value: "$(params.pipeline_image)"
+        - name: git_repo_url
+          value: $(params.git_repo_url)
+        - name: commit_sha
+          value: $(params.git_commit)
+        - name: description
+          value: "Pull request for operator successfully passed"
```

Comment:
The message in the hosted pipeline should be related to the operator test status. So something like "Operator tests pipeline has been successful". The same thing should be updated for failed status.